architecture/azure/processes.adoc

```diff
@@ -0,0 +1,51 @@
+[[azure-processes]]
+= Processes
+//openshift-aro 3.11 specific
+
+This section presents the summary of how Azure Red Hat OpenShift specific
+processes work. These descriptions do not go into all edge cases, and are
+limited to presenting the expected flow. Edge case handling can be observed
+in the source code repository for Azure Red Hat OpenShift.
+
+== Cluster update
+A new release can contain new Virtual Machine images or container images.
+It is done in the background without any customer involvement or any
+interruption to customer applications running on the cluster.
+All actions are done by the Azure Red Hat OpenShift Resource Provider.
+
+=== Preparation
+
+After the resource provider switches the cluster into the appropriate
+provisioning state it performs a backup of Etcd. Then generates all required
+configuration and template files and places them in the appropriate storage
+accounts.
+
+=== Master updates
+The master nodes are updated first. The Resource Provider checks if the current
+hash of master-000000 matches the desired hash value. If it does, the node is
+skipped and not updated. If it doesn't the node is drained and stopped, then
```

Comment:
s/it doesn't/values does not match,/
s/node/master/
s/drained and stopped, then/drained, stopped, and/